README.md

```diff
@@ -1,9 +1,270 @@
-📖 [Hello World Repository] 
+<a name="readme-top"></a>
 
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
 
+Modify this file to match your project and remove sections that don't apply.
 
-**[Hello World Repository]** is a static html and css page made as the first project for microverse.
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
 
--To run this application clone it into your local repository and run the index.html file
--It can be viewed on all supported browsers, namely: Google Chrome, Microsoft Edge... etc
+OPTIONAL SECTIONS:
+- FAQ
 
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello_World_Repo] <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
+
```

Comment:
- [ ] Please remove the default comment from your README e.g. 
 
 > Describe your project in 1 or 2 sentences.

> Describe the tech stack and include only the relevant sections that apply to your project.

> Write a message to encourage readers to support your project  

> Describe 1 - 3 features you will add to the project.   

,etc...  

They were used in the template to explain to the user the purpose of each section. You can go ahead and remove it from your README